tests/parametric/test_128_bit_traceids.py

```diff
@@ -158,6 +162,7 @@ def test_datadog_128_bit_propagation_and_generation(self, test_agent, test_libra
         assert trace_id == 1234567890123456789
         assert int(headers["x-datadog-trace-id"], 10) == trace_id
         assert dd_p_tid is None
+        assert "_dd.p.tid=" not in headers["x-datadog-tags"]
```

Comment:
This (and similar additions) isn't wrong but it seems redundant with the previous test.